.github/NativeBuildReport.kts

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("jcenter","https://jcenter.bintray.com/")
+@file:MavenRepository("maven-central","https://repo.maven.apache.org/maven2/")
+@file:DependsOn("org.kohsuke:github-api:1.101")
+
+
+import org.kohsuke.github.*
+import java.io.File
+import java.io.IOException
+import java.util.concurrent.TimeUnit
+
+val token = args[0]; 
+val status = args[1];
+
+val ISSUE_NUMBER=6717
```

Comment:
I would make the issue number a parameter so that we can easily reuse this script for other reports.

Reply:
Done